scripts/check-baseline.sh

```
@@ -0,0 +1,29 @@
+echo "1. Running bertie tests..."
+cd /home/raditha/csi/Antikythera/bertie
+if mvn test -q; then
+  echo "✅ Bertie tests passed (136 tests)"
+else
+  echo "❌ Bertie tests failed"
+  exit 1
+fi
+
+echo ""
+echo "2. Running test-bed baseline tests..."
+cd test-bed
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)